samples/svgc/Program.cs

```diff
@@ -42,6 +43,14 @@ public SMP.SKImage LoadImage(Stream stream)
             Height = image.Height
         };
     }
+
+    public List<(string text, float advance, SMP.SKTypeface? typeface)>
+        FindTypefaces(string text, SMP.SKPaint paintPreferredTypeface)
+    {
+        // TODO: How can font fallback be done in a code generator?
```

Comment:
That is issue with generator not working with SkiaSharp native assets for platforms, so probably not possible or very hard requiring some package reference hacks.